src/header/header-buttons.scss

```diff
@@ -0,0 +1,346 @@
+      }
+
+      // Blue text
+      .iui-button-label {
+        @include themed {
+          color: t(iui-color-foreground-primary);
+        }
+      }
+
+      &,
+      &:hover {
+        // Blue background and darker blue bottom line
+        @include themed {
+          box-shadow: 0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+          background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
+        }
+
+        .iui-header-breadcrumb-button-icon {
+          @include themed {
+            fill: t(iui-icons-color-primary);
+            background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
+          }
+        }
+      }
+
+      // Disabled state
+      &.iui-disabled,
+      &:disabled,
+      &[aria-disabled='true'] {
+        .iui-button-label {
+          @include themed {
+            color: t(iui-icons-color-actionable-disabled);
+          }
+        }
+
+        .iui-button-icon,
+        svg {
+          @include themed {
+            fill: t(iui-icons-color-actionable-disabled);
+          }
+        }
+
+        .iui-header-breadcrumb-button-icon {
+          @include themed {
+            background-color: rgba(t(iui-color-foreground-body-rgb), 0.05);
+          }
+        }
+      }
+    }
+
+    .iui-disabled.iui-header-breadcrumb-button-dropdown:hover {
+      background-color: transparent;
+    }
+  }
+}
+
+@mixin iui-header-breadcrumb-button-label {
+  font-family: $iui-font-family;
+  font-size: $iui-font-size;
+  line-height: $iui-line-height;
+  text-align: left;
+  overflow: hidden;
+  transition: color $iui-speed-fast ease-out;
+  @media (max-width: 768px) {
+    margin-left: 0;
+  }
+
+  &-name,
+  &-description {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  &-description {
+    height: $iui-baseline * 2;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out, opacity $iui-speed-fast ease-out;
+    }
+    @include themed {
+      opacity: t(iui-opacity-3);
+    }
+  }
+
+  @at-root {
+    .iui-slim .iui-header-breadcrumb-button-label-description {
+      @include iui-display-none;
+    }
+
+    .iui-active .iui-header-breadcrumb-button-label {
+      @include themed {
+        color: t(iui-color-foreground-primary);
+      }
+    }
+
+    .iui-disabled .iui-header-breadcrumb-button-label {
+      @include themed {
+        color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+      }
+    }
+  }
+}
+
+@mixin iui-header-breadcrumb-button-icon($size: $iui-icons-large, $border-radius: $iui-border-radius) {
+  width: $size;
+  height: $size;
+  border-radius: $border-radius;
+  will-change: transform;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: transform $iui-speed-fast ease-out;
+  }
+
+  @at-root {
+    .iui-disabled,
+    :disabled,
+    [aria-disabled='true'] {
+      .iui-header-breadcrumb-button-icon {
```

Comment:
If you remove nesting from the place that this mixin is included, then you can just use `&` instead of this line.

Every time you type out the name of a class, you should feel some "tingling" 😄